bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -512,7 +541,7 @@ def _nomination_to_string(self, nomination_object: dict) -> str:
                 {entries_string}
 
                 End date: {end_date}
-                Unwatch reason: {nomination_object["end_reason"]}
+                Unnominate reason: {nomination_object["end_reason"]}
```

Comment:
Would `Unnomination reason:` rather than `Unnominate reason:` be slightly better english? I'm not actually sure

Reply:
Yes, that would be correct. Will update later